Cannot find module '@babel/core'

**Describe the bug**

running webpack after `webpack-cli init` results in error `Cannot find module '@babel/core`

**What is the current behavior?**

running webpack after `webpack-cli init` results in error `Cannot find module '@babel/core`

**To Reproduce**

Steps to reproduce the behavior:
1. `npx webpack-cli init`
2. `Will your application have multiple bundles?` answer: n
3. `Which module will be the first to enter the application?` answer: ./src/bg
4. `Which folder will your generated bundles be in?` answer: dist
5. `Will you be using ES2015?` answer: Y
6. `Will you use one of the below CSS solutions?` answer: SASS
7. observe: `Congratulations! Your new webpack configuration file has been created!`
8. rename generated `webpack.dev.js` to `webpack.config.js`
9. fix generated `webpack.config.js` (see issue #675)
10. `npx webpack`
11. observe error `Cannot find module '@babel/core'`

**Expected behavior**

no error when running webpack after `webpack-cli init`

**Screenshots**

N/A

**Please paste the results of `webpack-cli info` here, and mention other relevant information**

  System:
    OS: Windows 10
    CPU: x64 Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  Binaries:
    npm: 5.3.0 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    uglifyjs-webpack-plugin: ^2.0.1 => 2.0.1
    webpack: ^4.24.0 => 4.24.0
    webpack-cli: ^3.1.2 => 3.1.2

**Additional context**

N/A

--- a/MIGRATE.md
+++ b/MIGRATE.md
@@ -57,7 +57,7 @@ module.exports = {
 		exclude: /node_modules/,
 		loader: 'babel',
 		query: {
-		  presets: ['env']
+		  presets: ['@babel/preset-env']
 		}
 	  },
 	  {
@@ -140,7 +140,7 @@ module.exports = {
 		  loader: 'babel-loader'
 		}],
 		options: {
-		  presets: ['env']
+		  presets: ['@babel/preset-env']
 		}
 	  },
 	  {

--- a/packages/generators/init-generator.ts
+++ b/packages/generators/init-generator.ts
@@ -151,9 +151,9 @@ export default class InitGenerator extends Generator {
 						getBabelPlugin(),
 					);
 					this.dependencies.push(
-						"babel-core",
 						"babel-loader",
-						"babel-preset-env",
+						"@babel/core",
+						"@babel/preset-env",
 					);
 				}
 			})

--- a/packages/generators/utils/module.ts
+++ b/packages/generators/utils/module.ts
@@ -24,7 +24,7 @@ export default function(): IModule {
 			],
 			presets: [
 				[
-					"'env'",
+					"'@babel/preset-env'",
 					{
 						"'modules'": false,
 					},

--- a/packages/migrate/loaders/__testfixtures__/loaders-7.input.js
+++ b/packages/migrate/loaders/__testfixtures__/loaders-7.input.js
@@ -6,7 +6,7 @@ export default {
                 exclude: /(node_modules)/,
                 loader: 'babel-loader',
                 query: {
-                    presets: ['env']
+                    presets: ['@babel/preset-env']
                 }
             }
         ]

